client/post-editor/editor-drawer/taxonomies.jsx

```diff
@@ -34,26 +35,29 @@ function EditorDrawerTaxonomies( { siteId, postType, postTerms, taxonomies } ) {
 						<Accordion
 							key={ taxonomy.name }
 							title={ taxonomy.label }
-							icon={ <Gridicon icon="category" /> }
+							icon={ <Gridicon icon="folder" /> }
 						>
-							Work in Progress
-						</Accordion>
-					);
-				} else {
-					return (
-						<Accordion
-							key={ taxonomy.name }
-							title={ taxonomy.label }
-							icon={ <Gridicon icon="tag" /> }
-						>
-							<TermTokenField
-								postTerms={ postTerms }
+							<TermSelector
+								selectedTermIds={ postTerms ? map( postTerms[ taxonomy.name ], 'ID' ) : [] }
 								taxonomyName={ taxonomy.name }
-								taxonomyLabel={ taxonomy.label }
 							/>
 						</Accordion>
 					);
 				}
+
+				return (
+					<Accordion
+						key={ taxonomy.name }
+						title={ taxonomy.label }
+						icon={ <Gridicon icon="tag" /> }
+					>
+						<TermTokenField
+							postTerms={ postTerms }
```

Comment:
@nylen : Aside, maybe `postTerms` isn't the best name for the prop, since it seems to unnecessarily tie the field to being used only the context of editing a post. What about something with `selected` instead?